scripts/Run-Angular-Locally.ps1

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env powershell
```

Comment:
I think I can combine Angular, React and ReactRedux into `Run-Spa-Locally.ps1` and just pass it the name of the SPA.

Reply:
By the way, change the shebang to `#!/usr/bin/env pwsh -c`. They renamed powershell => pwsh.